src/olympia/amo/management/commands/fix_licenses.py

```diff
@@ -0,0 +1,77 @@
+from django.core.management.base import BaseCommand
+from django.utils.translation import activate
+
+from olympia.amo.log import CHANGE_LICENSE
+from olympia.versions.models import License
+from olympia.addons.models import Addon
+from olympia.devhub.models import ActivityLog
+from olympia.translations.models import Translation
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        activate('en-us')
+
+        mpl11 = License.objects.get(pk=5)
+
+        assert str(mpl11.name) == 'Copyright Jason Savard', 'wrong license'
+        assert not mpl11.text
+
+        # set on_form explicitly to False to make sure it's not
+        # visible and set `builtin` to 1 to make sure it's filtered
+        # everywhere.
+        mpl11.on_form = False
+        mpl11.builtin = 1
+        mpl11.save()
+
+        search = '"versions.license": 5}'
```

Comment:
It's more generic but also matches `504` and other ids so I made this as strict as possible. We only know about 3 add-ons that actually match the description but I wanted this to be as testable on stage as possible.